src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -885,6 +886,43 @@ def detect(
                 how='diagonal',
             )
 
+    def compute_event_properties(
+            self,
+            event_properties: str | tuple[str, dict[str, Any]]
+            | list[str | tuple[str, dict[str, Any]]],
+            name: str | None = None,
+    ) -> None:
+        """Calculate an event property for and add it as a column to the event dataframe.
```

Comment:
I guess there's a word missing here?

maybe reformulate it like this and split it in two sentences?

```
Calculate event properties for given events.

The calculated event properties are added as columns to :py:attr:`~pymovements.gaze.GazeDataFrame.events`.
```